base_classes/NXbeam.nxdl.xml

```diff
@@ -43,84 +56,167 @@
     <field name="incident_energy" type="NX_FLOAT" units="NX_ENERGY">
         <doc>Energy on entering beamline component</doc>
         <dimensions rank="1">
-            <dim index="1" value="i"/>
+            <dim index="1" value="m"/>
         </dimensions>
     </field>
     <field name="final_energy" type="NX_FLOAT" units="NX_ENERGY">
         <doc>Energy on leaving beamline component</doc>
         <dimensions rank="1">
-            <dim index="1" value="i"/>
+            <dim index="1" value="m"/>
         </dimensions>
     </field>
     <field name="energy_transfer" type="NX_FLOAT" units="NX_ENERGY">
         <doc>Energy change caused by beamline component</doc>
         <dimensions rank="1">
-            <dim index="1" value="i"/>
+            <dim index="1" value="m"/>
```

Comment:
This one I don't know if it should be `value="m"` or `value=nP`.  Thoughts?

Reply:
m